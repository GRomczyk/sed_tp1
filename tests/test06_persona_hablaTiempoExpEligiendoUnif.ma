[Top]
%En este test esperamos que la persona hable en tiempo exponencial de media 60 segundos, eligiendo uniformemente el puerto de salida
components : P1@Persona
out : out_port1 out_port2 out_port3 out_port4 out_port5
in : in_port1 in_port2 in_port3 in_port4 in_port5 

Link : in_port1 inPort1@P1
Link : in_port2 inPort2@P1
Link : in_port3 inPort3@P1
Link : in_port4 inPort4@P1
Link : in_port5 inPort5@P1

Link : outPort1@P1 out_port1
Link : outPort2@P1 out_port2
Link : outPort3@P1 out_port3
Link : outPort4@P1 out_port4
Link : outPort5@P1 out_port5


[P1]
p : 1.0
influenciabilidad : 0.0
poder_convencimiento : 0.2
tiempo_sillon : 1.0
grado : 5
periodo_medio : 60
