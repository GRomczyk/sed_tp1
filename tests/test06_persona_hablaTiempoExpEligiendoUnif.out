00:00:19:397:0.865295 out_port4 [1, 0.2]
00:02:16:812:0.599731 out_port4 [1, 0.2]
00:04:35:454:0.285767 out_port1 [1, 0.2]
00:06:42:454:0.553345 out_port3 [1, 0.2]
00:06:57:249:0.452881 out_port4 [1, 0.2]
00:07:13:606:0.87912 out_port5 [1, 0.2]
00:08:21:240:0.203339 out_port2 [1, 0.2]
00:10:13:776:0.260406 out_port2 [1, 0.2]
00:10:18:930:0.0833778 out_port2 [1, 0.2]
00:12:48:933:0.173832 out_port1 [1, 0.2]
00:13:08:007:0.713566 out_port5 [1, 0.2]
00:13:14:865:0.641438 out_port1 [1, 0.2]
00:13:27:520:0.491993 out_port4 [1, 0.2]
00:13:39:448:0.717106 out_port1 [1, 0.2]
00:13:44:233:0.318867 out_port1 [1, 0.2]
00:14:02:151:0.93285 out_port5 [1, 0.2]
00:14:41:713:0.834492 out_port5 [1, 0.2]
00:14:52:122:0.120182 out_port5 [1, 0.2]
00:15:31:282:0.711857 out_port2 [1, 0.2]
00:17:38:573:0.766178 out_port3 [1, 0.2]
00:18:48:251:0.623173 out_port5 [1, 0.2]
00:18:53:302:0.306744 out_port2 [1, 0.2]
00:19:59:100:0.113079 out_port5 [1, 0.2]
00:20:17:313:0.702343 out_port1 [1, 0.2]
00:21:21:463:0.063549 out_port2 [1, 0.2]
00:21:23:750:0.450039 out_port1 [1, 0.2]
00:22:02:103:0.783108 out_port4 [1, 0.2]
00:24:08:313:0.286831 out_port2 [1, 0.2]
00:24:18:216:0.00115585 out_port3 [1, 0.2]
00:25:24:123:0.754696 out_port4 [1, 0.2]
00:28:51:505:0.171566 out_port2 [1, 0.2]
00:29:05:040:0.261608 out_port4 [1, 0.2]
00:29:22:369:0.934017 out_port1 [1, 0.2]
00:29:53:722:0.0234947 out_port5 [1, 0.2]
00:31:08:817:0.71954 out_port3 [1, 0.2]
00:31:38:218:0.816433 out_port2 [1, 0.2]
00:32:28:591:0.249691 out_port5 [1, 0.2]
00:33:39:013:0.308346 out_port2 [1, 0.2]
00:36:18:344:0.569576 out_port2 [1, 0.2]
00:36:51:359:0.00494003 out_port1 [1, 0.2]
00:40:22:327:0.137508 out_port5 [1, 0.2]
00:40:23:294:0.719807 out_port3 [1, 0.2]
00:41:01:424:0.080555 out_port2 [1, 0.2]
00:41:47:090:0.482014 out_port3 [1, 0.2]
00:44:42:074:0.636311 out_port5 [1, 0.2]
00:46:29:888:0.784809 out_port4 [1, 0.2]
00:47:07:289:0.542194 out_port2 [1, 0.2]
00:48:20:416:0.587421 out_port2 [1, 0.2]
00:50:37:100:0.113056 out_port2 [1, 0.2]
00:51:26:105:0.114674 out_port2 [1, 0.2]
00:51:41:084:0.657001 out_port2 [1, 0.2]
00:52:38:120:0.611073 out_port1 [1, 0.2]
00:53:08:018:0.0496712 out_port3 [1, 0.2]
00:53:35:036:0.76894 out_port5 [1, 0.2]
00:56:42:650:0.867085 out_port4 [1, 0.2]
00:56:44:739:0.557199 out_port3 [1, 0.2]
